### ISSUE TITLE
Meeting WCAG 2.1

### ISSUE THREAD

**Issue by jennpb** [wellcomecollection/wellcomecollection.org#3590]
Investigate what we would need to do to meet the new specs.

http://www.ashleysheridan.co.uk/blog/New+Features+in+Web+Content+Accessibility+Guidelines+2.1+and+What+You+Need+to+Do

**Comment by jennpb** [comment]
Also of interest, from the MCG list. While we are technically exempt from the incoming regulations, we should endeavour to meet or exceed:

> Dear MCGers,
> 
> I don't think it's been mentioned here, but new regulations seem to have come into effect from September (http://www.legislation.gov.uk/uksi/2018/952/introduction/made), applicable from next September.
> 
> Reading the explanatory memo, it seems that meeting WCAG 2.1 at level AA should suffice. I'd be interested to know if anyone's explored the exceptions around heritage material (e.g. we're a long way from accurate transcriptions for images of handwritten text).
> 
> It's a while since I've looked at accessibility tools - does anyone have recommendations for browser plugins etc that can test for compliance? I know they're only the first step but it's a useful first step nonetheless.
> 
> More info from http://www.legislation.gov.uk/uksi/2018/952/pdfs/uksiem_20180952_en.pdf
> 'Part 3 of the instrument details that where public sector bodies’ websites meet the European accessibility standard EN 301 549 (which is being updated to map to the latest version of the international web accessibility standard, WCAG 2.1 to level AA) it will be presumed to meet the requirements above. For mobile applications these will be presumed to meet the requirements where they meet technical specifications adopted by the EU, or, where no technical specifications have been produced, where the mobile application meets European accessibility standard EN 301 549.'
> 
> An accessibility statement must also be published, which 'must explain which parts of the content of a website or mobile application are not accessible, and, where appropriate, provide links to accessible alternatives. It will also include a contact form (e.g. an email address) for citizens to report accessibility issues and request information that has been excluded from a website or mobile application in an accessible format, and a link to the enforcement procedure if they are not content with response received from the public sector body'.
> 

[According to GDS](https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps#accessibility-statements), Accessibility Statements aren't required until 2020, but I've put it on the [Now > Next > Future board in Trello](https://trello.com/b/IxGsTJ1E/now-next-future).

**Comment by gestchild** [comment]
# WCAG 2.1

- 17 new Criteria, of which 5 are AAA and of lower priority
- Of the remaining 12, at least 3 are either already satisfied or not applicable
- The following 9 A/AA criteria still left to investigate/fix:

1.3.4 Orientation (Level AA)
1.3.5 Identify Input Purpose (AA)
1.4.11 Non-Text Contrast (AA)
2.1.4 Character Key Shortcuts (A)
2.5.1 Pointer Gestures (A)
2.5.2 Pointer Cancellation (A)
2.5.3 Label in Name (A)
2.5.4 Motion Actuation (A)
4.1.3 Status Messages (AA)

## Already satisfied
1.4.10 Reflow (AA)
1.4.12 Text Spacing (AA)

## Not currently applicable
1.4.13 Content on Hover or Focus (AA)

## AAA (lower priority)
1.3.6 Identify Purpose (AAA)
2.2.6 Timeouts (AAA)
2.3.3 Animation from Interactions (AAA)
2.5.5 Target Size (AAA)
2.5.6 Concurrent Input Mechanisms (AAA)

**Comment by gestchild** [comment]
@jennpb no, I don't believe so
